templates/cluster-template-with-disk-offering.yaml

```diff
@@ -101,28 +110,34 @@ spec:
     matchLabels: null
   template:
     spec:
+      clusterName: "${CLUSTER_NAME}"
+      version: "${KUBERNETES_VERSION}"
```

Comment:
I prefer the attributes to be alphabetically sorted like they were. Would it be difficult to reorder them that way again? It might be an editor preference but it would reduce the diff count and keep the formatting consistent

Reply:
Agreed that making these more diff-able is good.  I dislike, though, having to go looking for, say, identity fields, hidden in the middle of the body, when reading manifests.  How strongly do you feel about this?

Reply:
that's a fair point - I'll allow it :)

Reply:
Lol, you two.